docs/tutorial_advanced.rst

```diff
@@ -281,7 +281,7 @@ The following test is a compile-only version of the :class:`MakefileTest` presen
 
 
 .. literalinclude:: ../tutorials/advanced/makefiles/maketest.py
-   :lines: 32-
+   :lines: 30-
```

Comment:
@line 167: `The compiler variables ...` should appear in the first box ?

Reply:
@line 253: lower and upper bounds`.`

Reply:
Not sure what you mean on the first comment.

Reply:
I would expect to see:
`make -j 1 CPPFLAGS="-DELEM_TYPE=float" CC=cc CXX=CC` 
?

Reply:
@jgphpc You are right here. @jjotero he means that the emitted line is not correct as we show it in the tutorial.